tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/TemporalPassImpl.scala

```diff
@@ -31,14 +31,15 @@ class TemporalPassImpl @Inject() (
   override def execute(tlaModule: TlaModule): PassResult = {
     logger.info("  > Rewriting temporal operators...")
 
-    val newModule = options.get[List[String]]("checker", "inv") match {
+    val temporalProps = options.get[List[String]]("checker", "temporal")
+    val newModule = temporalProps match {
       case None =>
-        logger.info("  > No invariants specified, nothing to encode")
+        logger.info("  > No formula specified, nothing to encode")
```

Comment:
"formula" is maybe a bit ambiguous to understand what went wrong, shall we say
```suggestion
        logger.info("  > No temporal property specified, nothing to encode")
```